src/components/sidebar/Sidebar.tsx

```diff
@@ -48,46 +65,37 @@ const Menu = () => {
   useEffect(() => {
     const interval = setInterval(async () => {
       const newComponentsList = await API.get('/refresh').then(res => res.data)
+      dispatch.customComponents.updateCustomComponents(newComponentsList)
+
       const componentDiffs = getObjectDiff(newComponentsList)
       componentDiffs.deletedComponents.map(async component => {
-        const response = await API.post('/delete-file', {
+        const _ = await API.post('/delete-file', {
           path: customComponents[component],
         })
+        autoselectId()
```

Comment:
Why auto select after each component deletion. Can we do that after all components deletion